src/tests/test_PackageNameVersion.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+import sys
+sys.path.append('./src/')
+from Package import PackageNameVersion
+
+class Test_PackageNameVersion:
+    def test_validInputsNoVersion(self):
```

Comment:
I like that you use table driven tests but I have a question: why haven't you used [pytest parametrisation feature](https://docs.pytest.org/en/7.4.x/how-to/parametrize.html)?